.travis.yml

```diff
@@ -27,7 +27,7 @@ matrix:
           php: 7.1
           # Disable code coverage until https://github.com/leanphp/phpspec-code-coverage/pull/38 is released
           # DEPENDENCIES="leanphp/phpspec-code-coverage:^4.2" TEST_COMMAND="composer test-ci"
-          env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" COVERAGE=true TEST_COMMAND="composer test" PULI_VERSION=1.0.0-beta9
+          env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" COVERAGE=true TEST_COMMAND="composer test" DEPENDENCIES="php-http/httplug:^2.0" PULI_VERSION=1.0.0-beta9
```

Comment:
this means we no longer test with httplug 1 at all, as all the other builds install newest.